yarpl/include/yarpl/flowable/Subscribers.h

```diff
@@ -76,14 +76,28 @@ class Subscribers {
     }
 
     void onNext(T value) override {
-      next_(std::move(value));
+      if (userError_) {
```

Comment:
You don't need `userError_`.  You can check if `Subscriber<T>::subscription()` is null or not.

Reply:
If the user provides a Subscriber, then I believe that they need to write a Subscriber that handles this issue, at least for v1.
If the user only provides a lambda, then we should handle this issue.

Reply:
I want to be explicit about the `userError_` in here. 
The subscription() might be null for various reasons. I want to be able to differentiate this specific case. Because of this I decided to keep that variable.

Reply:
In theory, it should only be nullptr if the Subscriber hasn't yet called `onSubscribe()` (in which you wouldn't be calling these other onFoo() methods), or if the `Subscriber` is terminated, which gives you exactly what you want.

Reply:
ok.